### PR TITLE
fix safe_late_task related issues

### PR DESCRIPTION
**Pull request by imzhenyu**
fix problem mentioned in https://github.com/imzhenyu/rDSN/issues/34

### CODE REVIEW

File: include/dsn/cpp/clientlet.h [hunk 1]
```diff
@@ -103,7 +103,7 @@ namespace dsn
         }
 
         template<typename THandler>
-        safe_late_task<THandler>* create_late_task(
+        inline safe_late_task<THandler>* create_late_task(
```
Comment (qinzuoyan) [inline include/dsn/cpp/clientlet.h:106]: why template method need inline key word?

Comment (imzhenyu) [inline include/dsn/cpp/clientlet.h:106]: This is orthogonal to template - simply want to make it inlined to boost the performance.
